components/script/layout_wrapper.rs

```diff
@@ -710,8 +732,8 @@ pub struct ServoThreadSafeLayoutNode<'ln> {
     /// The wrapped node.
     node: ServoLayoutNode<'ln>,
 
-    /// The pseudo-element type, with (optionally),
-    /// an specified display value to override the stylesheet.
+    /// The pseudo-element type, with (optionally)
+    /// a specified display value to override the stylesheet.
```

Comment:
arrg, Bobby the dictionary strikes again to correct my typos! :P 

Thanks for doing this.